projects/map-ol/src/lib/map-ol.component.ts

```diff
@@ -229,11 +231,26 @@ export class MapOlComponent implements OnInit, AfterViewInit, AfterViewChecked,
       case WmtsLayertype:
         this.updateWmtsLayerParamsWith(oldLayer, newLayer as WmtsLayer);
         break;
+      case GeojsonLayertype:
+        this.updateGeojsonLayerParamsWith(oldLayer, newLayer as VectorLayer);
+        break;
       default:
         break;
     }
   }
 
+  updateGeojsonLayerParamsWith(oldLayer: olLayer<any>, newGeojsonLayer: VectorLayer) {
+    if (newGeojsonLayer.data) {
+      const newSource = new VectorSource({
+        features: this.mapSvc.geoJsonToFeatures(newGeojsonLayer.data),
+        wrapX: false
```

Comment:
For the [wrapX](https://github.com/openlayers/openlayers/blob/v6.4.3/src/ol/source/Source.js#L139) attribute it would be good if we can get this from the old source.
e.g. 
```ts
wrapX: oldLayer.getSource()?.getWrapX() || false
```